### DISCUSSION TITLE
Links out from seqr gene page to other databases outdated/errors

### DISCUSSION THREAD

**Discussion by lwin2022** [broadinstitute/seqr#4018] [Q&A]
Links from the gene page out to monarch/decpher/uniprot result in errors.

The link out to decipher is outdated
![image](https://github.com/broadinstitute/seqr/assets/116073465/092f4e46-0050-4d5b-8d63-16d366c4ebbd)
Monarch does not seem to support searching by ENSEMBL ID
![image](https://github.com/broadinstitute/seqr/assets/116073465/1ae4cf86-a3dd-403c-86ca-a7fc0bf41d29)
The terms sent to Uniprot result in an error
![image](https://github.com/broadinstitute/seqr/assets/116073465/3994682f-fe0d-4e82-be52-0f9643f6a987)
I believe the correct format would be ENSG00000178409 AND (reviewed:true) AND (organism_id:9606)

Using a local install of seqr v1.0-7738cea8.
Is this something that we can amend in our local install?

Thanks

**Comment by hanars** [comment]
Please update to the latest version of seqr to resolve these issues
